integration-test-common/src/test/java/org/apache/gravitino/integration/test/util/AbstractIT.java

```diff
@@ -58,61 +58,61 @@
 import org.apache.gravitino.server.web.JettyServerConfig;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.CsvSource;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testcontainers.shaded.org.awaitility.Awaitility;
 
 @ExtendWith({PrintFuncNameExtension.class, CloseContainerExtension.class})
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Can this annotation be inherited by subclasses?

Reply:
Yes.  Please see `SparkHiveCatalogIT33` and `SparkHiveCatalogIT`

Reply:
If that, we call remove all the  annotation like `@TestInstance(TestInstance.Lifecycle.PER_CLASS)` in the subclasses.

Reply:
OK, I have removed the redundant annotations.